mpas_analysis/ocean/ohc_timeseries.py

```diff
@@ -62,10 +62,11 @@ def ohc_timeseries(config, streamMap=None, variableMap=None):
     # Note: input file, not a mesh file because we need dycore specific fields
     # such as refBottomDepth and namelist fields such as config_density0, as
     # well as simulationStartTime, that are not guaranteed to be in the mesh file.
-    try:
-       inputfile = streams.readpath('restart')[0]
-    except ValueError, e:
-       raise ValueError('No MPAS-O restart file found: need at least one rst file for OHC calculation')
+    restartFileNames = streams.readpath('restart')
+    if len(restartFileNames) > 0:
+        inputfile = restarFileNames[0]
```

Comment:
This is entirely my fault, but there's a typo here.  `restarFileNames` should be `restartFileNames`.  Please fix this and test to make sure it works.